src/Common/src/Interop/Windows/kernel32/Interop.CreateNamedPipeClient.Uap.cs

```diff
@@ -32,7 +32,7 @@ internal partial class Kernel32
             parameters.dwSize = (uint)Marshal.SizeOf<Interop.Kernel32.CREATEFILE2_EXTENDED_PARAMETERS>();
 
             parameters.dwFileAttributes = (uint)dwFlagsAndAttributes & 0x0000FFFF;
-            parameters.dwSecurityQosFlags = (uint)dwFlagsAndAttributes & 0x000F0000;
+            parameters.dwSecurityQosFlags = (uint)dwFlagsAndAttributes & 0x001F0000;
```

Comment:
Short of defining enums and consts and the like, could you add a comment about what 0x001F0000 means?

Reply:
Hum, didn't like my first pass at the comment. I will edit it a little when I add the test.